openapi.yaml

```diff
@@ -444,6 +481,26 @@ components:
           schema:
             $ref: '#/components/schemas/eventV1PostBody'
       required: true
+    eventPostV1RadioText:
+      description: |
+        New event to be distributed to subscribers.
+        The Eventhub format validation expects only a subset of these variables as minimum set. All other fields are technically optional, but **highly encouraged** to be included, so a best-possible metadata exchange is possible.
+        The subset is defined in the list of required fields of Schemas `eventV1PostRadioTextBody`, resulting in this body:
+        ```json
+        {
+          "event": "music",
+          "start": "2020-01-19T06:00:00+01:00",
+          "validUntil": "2026-01-19T06:00:00+01:00",
```

Comment:
`validUntil` example should be greater than `start`?

Reply:
why not `end` or `length` – that would be more consistent with the existing event types, c.f. https://swrlab.github.io/ard-eventhub/#/docs/TYPES